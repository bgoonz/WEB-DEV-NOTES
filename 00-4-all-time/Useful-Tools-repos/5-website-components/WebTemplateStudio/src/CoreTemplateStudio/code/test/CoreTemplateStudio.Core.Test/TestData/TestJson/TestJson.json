{
  "TestKey": {
    "jkl": "101112"
  }
}