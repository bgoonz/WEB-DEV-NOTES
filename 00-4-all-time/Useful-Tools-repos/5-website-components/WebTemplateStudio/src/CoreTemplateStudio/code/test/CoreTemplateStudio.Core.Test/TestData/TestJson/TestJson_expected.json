{
  "TestKey": {
    "jkl": "101112",
    "abc": "123",
    "def": "456",
    "ghi": "789"
  }
}